2.overview_and_terminology.md

```diff
@@ -20,7 +20,7 @@ This specification proposes a model that defines cloud native applications as fo
 
 > A cloud native application is a collection of interrelated, but discrete _components_ (services, tasks, workers) that, when coupled with configuration and instantiated in a suitable runtime, together accomplish a unified functional purpose.
 
-This specification defines the following application constructs for application developers and application operators to design and operate applications:
+This specification defines the following application constructed for application developers and application operators to design and operate applications:
```

Comment:
why use passive voice?

Reply:
I think the meaning is that 

*This specification defines the following application **(that is)** constructed for application developers and application operators to design and operate applications:*

Reply:
constructs here is a `noun`